cgs_vmc/training.py

```diff
@@ -459,7 +469,7 @@ def build_opt_ops(
     # )
 
     loss = tf.reduce_mean(
-        tf.squared_difference(psi, psi_target)
+        tf.squared_difference(psi, psi_target * hparams.scale)
```

Comment:
I would move scaling up in the code to improve readability. One naturally would expect target to be target, not a target before scaling.